support-frontend/assets/helpers/timeBoundedCopy/timeBoundedCopy.tsx

```diff
@@ -13,23 +13,11 @@ type TimeBoundCopy = {
 export type TimedCopyCollection = Partial<Record<LandingPage, TimeBoundCopy[]>>;
 
 const timedCopy: TimedCopyCollection = {
-	digitalSubscription: [
+	newspaper: [
 		{
 			startShowingOn: '2022-04-14',
 			stopShowingOn: '2022-06-09',
-			copy: (
-				<>
-					<p>
-						<strong>With two innovative apps and ad-free reading,</strong> a
-						digital subscription gives you the richest experience of Guardian
-						journalism. It also sustains the independent reporting you love.
-					</p>
-					<p>
-						Plus, for a limited time, get inspiration for new ideas to improve
-						your life in our exclusive special edition, Self and Wellbeing.
-					</p>
-				</>
-			),
+			copy: <></>,
```

Comment:
What’s this? If `stopShowingOn` being in the past means it’s no longer used, can we just get rid of it altogether?

Reply:
We could definitely get rid of this variable but in the past we've kept an example in of how to structure the timed copy data. This is a helper (not currently in use) to allow us to easily make copy changes on our landing pages.

Reply:
Ah yes, that makes sense – thanks for the explanation :)